.travis-build-without-test.sh

```diff
@@ -24,15 +24,16 @@ if [[ "$SLUGOWNER" == "" ]]; then
 fi
 
 ## Build annotation-tools (Annotation File Utilities)
+WGETCMD="wget -q --spider"
```

Comment:
How about moving this before the annotations-tools comment, b/c this isn't just used for AFU.
How about naming it 'GITEXISTS' as a more descriptive name of what we want to use this for and a short comment.
ls-remote is also used in `.travis-build.sh`, so please also update that file.